README.md

```diff
@@ -73,4 +73,5 @@ No Modules.
 | aws\_kms\_alias\_arn | The Amazon Resource Name (ARN) of the key alias. |
 | aws\_kms\_alias\_name | The display name of the alias. |
 | aws\_kms\_key\_arn | The Amazon Resource Name (ARN) of the key. |
+| aws\_kms\_key\_id | The id of the key
```

Comment:
```suggestion
| aws\_kms\_key\_id | The globally unique identifier for the key.
```

Usually try to copy the value from the terraform docs if its just a pass through.